tests/integration/routes/institutions/dashboard/-components/institutional-users-list/component-test.ts

```diff
@@ -0,0 +1,80 @@
+        this.store = this.owner.lookup('service:store');
+        this.owner.register('service:router', OsfLinkRouterStub);
+    });
+
+    test('it renders and paginates', async function(assert) {
+        server.create('institution', { id: 'testinstitution' }, 'withInstitutionalUsers', 'withStatSummary');
+
+        this.set('modelTask', task(function *(this: TestContext, institutionId: string) {
+            return yield this.get('store').findRecord('institution', institutionId);
+        }));
+
+        const model = {
+            taskInstance: this.get('modelTask').perform('testinstitution'),
+        };
+
+        this.set('model', model);
+        await render(hbs`<Institutions::Dashboard::-Components::InstitutionalUsersList @model={{this.model}} />`);
+
+        assert.dom('[data-test-header-name]')
+            .exists({ count: 1 }, '1 name header');
+        assert.dom('[data-test-header-department]')
+            .exists({ count: 1 }, '1 departments header');
+        assert.dom('[data-test-header-public-projects]')
+            .exists({ count: 1 }, '1 public projects header');
+        assert.dom('[data-test-header-private-projects]')
+            .exists({ count: 1 }, '1 private projects header');
+
+        assert.dom('[data-test-item-name]')
+            .exists({ count: 10 }, '10 in the list with a name');
+        assert.dom('[data-test-item-department]')
+            .exists({ count: 10 }, '10 in the list with department');
+        assert.dom('[data-test-item-public-projects]')
+            .exists({ count: 10 }, '10 in the list with public project');
+        assert.dom('[data-test-item-private-projects]')
+            .exists({ count: 10 }, '10 in the list with private projects');
+
+        await click('[data-test-next-page-button]');
+
+        assert.dom('[data-test-item-name]')
+            .exists({ count: 5 }, '5 in the list with a name');
+        assert.dom('[data-test-item-department]')
+            .exists({ count: 5 }, '5 in the list with department');
+        assert.dom('[data-test-item-public-projects]')
+            .exists({ count: 5 }, '5 in the list with public project');
+        assert.dom('[data-test-item-private-projects]')
+            .exists({ count: 5 }, '5 in the list with private projects');
+    });
+
+    test('it sorts', async function(assert) {
+        server.create('institution', { id: 'test' }, 'withInstitutionalUsers', 'withStatSummary');
+        const model = { taskInstance: this.store.findRecord('institution', 'test') };
+        this.set('model', model);
+        await render(hbs`<Institutions::Dashboard::-Components::InstitutionalUsersList @model={{this.model}} />`);
+
+        assert.dom('[data-test-item-name]')
+            .exists({ count: 10 }, '10 users');
+
+        await click('[data-test-ascending-sort="department"]');
+        assert.dom('[data-test-item-department]')
+            .hasText('Architecture', 'Sorts by department ascendening');
```

Comment:
This isn't definitive (the department randomly assigned to the first institutional user could be `Architecture`). Better would be to set this up in a way where the first user's department would never be `Architecture` by default.